ntbs-ui-tests/Steps/Steps.cs

```diff
@@ -43,19 +43,39 @@ public void GivenIAmOnPage(string pageName)
         [When(@"I enter (.*) into '(.*)'")]
         public void WhenIEnterValueIntoFieldWithId(string value, string elementId)
         {
+            FindById(elementId).Click();
             FindById(elementId).SendKeys(value);
         }
 
+        [When(@"I wait")]
+        public void WhenIWait()
+        {
+            Thread.Sleep(2000);

```

Comment:
I added a lot of waits, because when debugging locally (and showing the chrome window) it kept not being able to access elements on the page. However, I just realised that when running in headless mode it's also fine locally without all these waits...
Should I check whether we can access the browser options during the tests, and only apply the waits if headlless mode is off? It's much eaiser to debug when seeing the browser window, so would be good if it works in all cases but not make running tests on jenkins unnecessarily slow

Reply:
yes, that sounds like a really good solution - I think it's the same problem i saw when running it not headless, but didn't know was wrong!